proto/spire/api/server/logger/v1/logger.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package spire.api.server.logger.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/server/logger/v1;loggerv1";
+
+import "google/protobuf/empty.proto";
```

Comment:
unused

Reply:
Thanks for the attention to detail.  Fixed in the next push for both the server and agent API protos.